test/e2e/common_test.go

```diff
@@ -296,6 +296,13 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 		var pathPrefix string
 		if !rootless.IsRootless() {
 			pathPrefix = "/run/podman/podman"
+			// FIXME: podman system service fails to start on Ubuntu 22.04 for
+			// some reason because of which we're creating /run/podman if it
+			// doesn't exist.
+			if _, err := os.Stat("/run/podman"); os.IsNotExist(err) {
+				mkdirErr := os.MkdirAll("/run/podman", 0755)
```

Comment:
Suggestion: `0700`, not 755. And @containers/podman-maintainers I think a better solution would be to understand why Ubuntu is doing this.

Reply:
Is the podman tmpfiles.d file installed correctly?
```
$ grep /run/podman /usr/lib/tmpfiles.d/podman.conf
D! /run/podman 0700 root root

```

Reply:
For podman CI, we remove all the files from the RPM then are suppose to do a `make install`.  After looking at the `Makefile`, we should be installing.   Checking if the logs give any clues why it's not there...

Reply:
Oh! So when I looked at the VM, I canceled the setup process - thinking (wrongly) this file was installed at VM image build time.  Checking again, the file is definitely there.  Maybe is there some systemd service that needs to be restarted/reloaded to pick up the file?  If so, I wonder how is/was this never a problem before :man_shrugging:

Reply:
I believe I follow what's happening here.  The entry in `podman.conf` is `D! /run/podman 0700 root root`, which means "Only create this directory on boot".  So the `make install`, nor any amount of service reloading will create this directory at test runtime.  Though, there should have been a podman package installed on the VM during the build process.  **THAT** should have placed `podman.conf` file such that the directory gets created on boot.  So maybe the Ubuntu podman package is broken then...

Reply:
@cevich fixed in the latest build. For whatever reason, debbuild can't seem to make sense of `%{_tmpfilesdir}` so I explicitly spelled out the path.

Reply:
Great, I still have an image build PR open, I'll trigger a re-build there...

Reply:
...[Images building here.](https://github.com/containers/automation_images/pull/140)

Reply:
@lsm5 I confirmed in a newly build Ubuntu container image that the `/usr/lib/tmpfiles.d/podman.conf` is there now.  Thanks :D